README.md

```diff
@@ -9,7 +9,7 @@ It works by generating a presigned URL that the Git LFS client can use to upload
 Git LFS S3 is available on RubyGems.
 
 ``` bash
-gem install git-lfs-s3
+gem install lsst-git-lfs-s3
```

Comment:
I have a mild objection to adding changes that will conflict with upstream but I can live with this.